docs/source/examples/Catch that asteroid!.ipynb

```diff
@@ -194,17 +165,17 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
+   "execution_count": 9,
    "metadata": {},
    "outputs": [
     {
      "data": {
       "text/plain": [
-       "(<Quantity [-2.76132873e+08, -1.71570015e+08, -1.09377634e+08] km>,\n",
-       " <Quantity [13.17478674, -9.82584125, -1.48126639] km / s>)"
+       "(<Quantity [-9.80228537e+07, -2.29262614e+08, -1.01451789e+08] km>,\n",
```

Comment:
The data here changed a lot, see my previous comment for a possible reason.

Reply:
I checked the Jupyter example after applying proper conversion to true anomaly and this solved immediately.